src/views/Tasks/TaskGroup/helmet.jsx

```diff
@@ -0,0 +1,13 @@
+import { Helmet } from 'react-helmet';
+import React, { Component } from 'react';
+
+export default class TestHelmet extends Component {
```

Comment:
I don't think you need to put that in its own file. You can just inline it in the code.